07-Modelling-Patchy-Particles/02-Simulating-a-System-of-Patchy-Particles.ipynb

```diff
@@ -0,0 +1,1909 @@
+{
```

Comment:
<p>The word vector is spelled as vecotr in one place in this block.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Tutorial authors should use https://pypi.org/project/jupyterlab-spellchecker/ or a alternative tool to spell check the notebook.